db/migrate/20230926132500_add_external_ids_to_events.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddExternalIdsToEvents < ActiveRecord::Migration[7.0]
+  def change
+    add_column :events, :external_customer_id, :string
```

Comment:
Rails/BulkChangeTable: You can use `change_table :events, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)